src/buffer/blocks.rs

```diff
@@ -217,7 +217,7 @@ impl<'slice, 'sample> Block<'slice, 'sample> {
     }
 
     /// Get a SIMD vector containing the channel data for a specific sample in this block. If `LANES
-    /// > channels.len()` then this will be padded with zeroes. If `LANES < channels.len()` then
+    /// \> channels.len()` then this will be padded with zeroes. If `LANES < channels.len()` then
```

Comment:
You have a formatter doing something weird here too. May want to look into that!

Reply:
This was a clippy warning actually, something about using a special character so I escaped it manually. I'll double back on this